scripts/HPC_chatbot_preprocessor/parsed_mds/generic/FAQ/FAQ_paragraph_6.txt

```diff
@@ -0,0 +1,37 @@
+$ setfacl -m u:otheruser:r dataset.txt
+$ ls -l dataset.txt
+-rwxr-x---+ 2 vsc40000 mygroup      40 Apr 12 15:00 dataset.txt
+For more information about chmod or setfacl, see
+Linux tutorial.
+Can I use multiple different SSH key pairs to connect to my VSC account?
+Yes, and this is recommended when working from different computers.
+Please see Adding multiple SSH public keys on how to do this.
+I want to use software that is not available on the clusters yet
+Please fill out the details about the software and why you need it in
+this form:
+https://www.ugent.be/hpc/en/support/software-installation-request.
+When submitting the form, a mail will be sent to hpc@ugent.be containing all the
+provided information. The HPC team will look into your request as soon
+as possible you and contact you when the installation is done or if
+further information is required. 
+Is my connection compromised? Remote host identification has changed
+On Monday 25 April 2022, the login nodes received an update to RHEL8.
```

Comment:
euhm, this is a section that should be removed (and then readded with the el9 variant).